hack/lib.sh

```diff
@@ -109,7 +109,7 @@ is_containerized() {
 
 containerize() {
   local cmd="$1"
-  local image="${CONTAINERIZE_IMAGE:-quay.io/kubermatic/util:2.2.0}"
+  local image="${CONTAINERIZE_IMAGE:-quay.io/kubermatic/util:2.3.1}"
```

Comment:
Why do we need this change?

Reply:
Correct

Reply:
https://github.com/kubermatic/kubermatic/pull/12694